### ISSUE TITLE
Add fontResource in addition to fontFamilyResource in compose module

### ISSUE THREAD

**Issue by Alex009** [icerockdev/moko-resources#464]
```kotlin
@Composable
expect fun fontResource(fontResource: FontResource, weight: FontWeight, style: FontStyle): Font
```

to use like this:
```kotlin
val MonumentExtended = FontFamily(
    Font(R.font.monument_extended_black, FontWeight.Black),
    Font(R.font.monument_extended_regular, FontWeight.Normal),
    Font(R.font.monument_extended_light, FontWeight.Light),
)
```
and
```kotlin
large = TextStyle(
    fontFamily = MonumentExtended,
    fontWeight = FontWeight.Black,
    fontSize = 24.sp,
    letterSpacing = 0.sp,
),
mediumBold = TextStyle(
    fontFamily = MonumentExtended,
    fontWeight = FontWeight.Bold,
    fontSize = 16.sp,
    letterSpacing = 0.sp,
),
mediumLight = TextStyle(
    fontFamily = MonumentExtended,
    fontWeight = FontWeight.Light,
    fontSize = 16.sp,
    letterSpacing = 0.sp,
),
small = TextStyle(
    fontFamily = MonumentExtended,
    fontWeight = FontWeight.Light,
    fontSize = 10.sp,
    letterSpacing = 0.sp,
),
```

**Comment by Alex009** [comment]
will be released in 0.23.0

api - `FontResource.asFont()`

**Comment by DATL4G** [comment]
@Alex009 is the `@Composable` annotation required?

I normally create / load a `FontFamily` like this into a `val` so it doesn't reload every time when I call it.
This does **not** require the `@Composable` annotation.

```kotlin
val Manrope: FontFamily by lazy(LazyThreadSafetyMode.NONE) {
    FontFamily(
        SharedRes.fonts.Manrope.`extra-light`.toComposeFont(FontWeight.ExtraLight),
        SharedRes.fonts.Manrope.`extra-light-italic`.toComposeFont(FontWeight.ExtraLight, FontStyle.Italic),

        SharedRes.fonts.Manrope.light.toComposeFont(FontWeight.Light),
        SharedRes.fonts.Manrope.`light-italic`.toComposeFont(FontWeight.Light, FontStyle.Italic),

        SharedRes.fonts.Manrope.regular.toComposeFont(FontWeight.Normal),
        SharedRes.fonts.Manrope.`regular-italic`.toComposeFont(FontWeight.Normal, FontStyle.Italic),

        SharedRes.fonts.Manrope.medium.toComposeFont(FontWeight.Medium),
        SharedRes.fonts.Manrope.`medium-italic`.toComposeFont(FontWeight.Medium, FontStyle.Italic),

        SharedRes.fonts.Manrope.`semi-bold`.toComposeFont(FontWeight.SemiBold),
        SharedRes.fonts.Manrope.`semi-bold-italic`.toComposeFont(FontWeight.SemiBold, FontStyle.Italic),

        SharedRes.fonts.Manrope.bold.toComposeFont(FontWeight.Bold),
        SharedRes.fonts.Manrope.`bold-italic`.toComposeFont(FontWeight.Bold, FontStyle.Italic),

        SharedRes.fonts.Manrope.`extra-bold`.toComposeFont(FontWeight.ExtraBold),
        SharedRes.fonts.Manrope.`extra-bold-italic`.toComposeFont(FontWeight.ExtraBold, FontStyle.Italic),
    )
}
```

And my `Typography` function then is a `@Composable`

```kotlin
@Composable
fun ManropeTypography(): Typography {
	return Typography(
		displayLarge = TextStyle(
			fontFamily = fontFamily,
        	fontWeight = FontWeight.Normal,
			...
		),
		...
	)
}
```
